simplematrixbotlib/match.py

```diff
@@ -111,26 +111,26 @@ def command(self, command=None):
             elif self.mention():
                 # the order is important here!
                 # note: we assume that body and formatted_body, if present, match as a workaraound of cleaning html
-                if self.event.body.startswith(self._disambiguated_name):
-                    self._body_without_prefix = self.event.body[len(self._disambiguated_name):]
-                elif self.event.body.startswith(self._display_name):
-                    self._body_without_prefix = self.event.body[len(self._display_name):]
-                elif self.event.body.startswith(self.room.own_user_id):
-                    self._body_without_prefix = self.event.body[len(self.room.own_user_id):]
-                elif self.event.formatted_body.startswith(self._pill):
-                    name = self.event.formatted_body[len(self._pill):self.event.formatted_body.index('</a>')]
+                id_matched = False
+                for id in (self._disambiguated_name, self._display_name, self.room.own_user_id):
+                    if self.event.body.startswith(id) and not id_matched:
+                        self._body_without_prefix = self.event.body[len(id):]
+                        id_matched = True
```

Comment:
`break` if a match was found

Reply:
Either way has the same functionality. It is the difference between an extra line vs checking a couple extra if statements. I have no preference